test_scripts/Defects/5_0/2448_SDL_does_not_clear_app_info_after_MASTER_RESET/2448_1_SDL_clears_app_info_after_MASTER_RESET.lua

```diff
@@ -0,0 +1,33 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2448
+--
+-- Reproduction Steps:
+-- HMI and SDL are started.
+-- Register and activate Application1.
+-- HMI sends notification OnExitAllApplication{reason = MASTER_RESET}.
+--
+-- Expected Behavior:
+-- All SDL data are cleaned and reset.
```

Comment:
@IGapchuk Please correct expectation according to check implemented in `common.HMISendToSDL_MASTER_RESET`. In this function only absence of `app_info.dat` is verified (not `All SDL data`).